src/test/package/nuget/NuGetTest/NuGetTest.sln

```diff
@@ -0,0 +1,179 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29025.244
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "TestApp", "TestApp\TestApp.vcxproj", "{A8BDBDE9-1A3D-4F5E-8668-9F6E84790D44}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "TestRuntimeComponent1", "TestRuntimeComponent1\TestRuntimeComponent1.vcxproj", "{E0EBBE54-C046-4611-B048-3CE893B1DF8A}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "TestRuntimeComponent2", "TestRuntimeComponent2\TestRuntimeComponent2.vcxproj", "{4BBB2DE7-4596-4DA6-A923-E65E838363B5}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "TestStaticLibrary1", "TestStaticLibrary1\TestStaticLibrary1.vcxproj", "{2158F418-CA97-4599-8103-EFC133850BAA}"
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "TestStaticLibrary2", "TestStaticLibrary2\TestStaticLibrary2.vcxproj", "{0011F69F-2363-4DFD-B02A-1E7E909BCE89}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{41E05048-C8B0-4AB7-BE1B-D8D53E60A5E4}"
+	ProjectSection(SolutionItems) = preProject
+		..\..\..\..\package\cppwinrt\nuget\Microsoft.Windows.CppWinRT.props = ..\..\..\..\package\cppwinrt\nuget\Microsoft.Windows.CppWinRT.props
+		..\..\..\..\package\cppwinrt\nuget\Microsoft.Windows.CppWinRT.targets = ..\..\..\..\package\cppwinrt\nuget\Microsoft.Windows.CppWinRT.targets
+	EndProjectSection
```

Comment:
may want to also drop the directory.build.* files in here too, to highlight them and as a convenience for maintenance